apps/example/detox.config.js

```diff
@@ -0,0 +1,93 @@
+const iosName = 'MobileStack'
+const derivedDataPath = 'ios/build'
+const sdk = 'iphonesimulator'
+
+module.exports = {
+  testRunner: {
+    args: {
+      $0: 'jest',
+      config: require.resolve('./e2e/jest.config.js'),
+      _: ['e2e'],
+    },
+    forwardEnv: true,
+  },
+  apps: {
+    'ios.release': {
+      type: 'ios.app',
+      binaryPath: `${derivedDataPath}/Build/Products/Release-${sdk}/${iosName}.app`,
+      build: `set -o pipefail && xcodebuild -workspace ios/${iosName}.xcworkspace -scheme ${iosName} -configuration Release -sdk iphonesimulator -derivedDataPath ${derivedDataPath} 2>&1 | npx excpretty ./`,
+    },
+    'ios.debug': {
+      type: 'ios.app',
+      binaryPath: `${derivedDataPath}/Build/Products/Debug-${sdk}/${iosName}.app`,
+      build: `set -o pipefail && xcodebuild -workspace ios/${iosName}.xcworkspace -scheme ${iosName} -configuration Debug -sdk iphonesimulator -derivedDataPath ${derivedDataPath} 2>&1 | npx excpretty ./`,
+    },
+    'android.debug': {
+      type: 'android.apk',
+      binaryPath: 'android/app/build/outputs/apk/debug/app-debug.apk',
+      build:
+        'pushd android && ./gradlew app:assembleDebug app:assembleAndroidTest -DtestBuildType=debug && popd',
+    },
+    'android.release': {
+      type: 'android.apk',
+      binaryPath: 'android/app/build/outputs/apk/release/app-release.apk',
+      build:
+        'pushd android && ./gradlew app:assembleRelease app:assembleAndroidTest -DtestBuildType=release && popd',
+    },
+  },
+  devices: {
+    simulator: {
+      type: 'ios.simulator',
+      device: {
+        type: 'iPhone SE (2nd generation)',
+      },
+    },
+    'simulator.15.0': {
+      type: 'ios.simulator',
+      device: {
+        type: 'iPhone SE (2nd generation)',
+        os: 'iOS 15.0',
```

Comment:
Upgrading the simulators is outside the scope of this PR. The latest version of Xcode does not include simulator support for iOS 15.0. The simulator will launch if it is available but cannot be controlled manually.

Reply:
Ah oops I must have copy pasted this incorrectly.

Reply:
ah no, this was the same config we had in the wallet for iOS: https://github.com/valora-inc/wallet/blob/c6c57b0ff3d3178419465016aa199e37f85be3ff/package.json#L369-L376

Reply:
Sorry for the confusion. I Was just commenting at some point we likely want to upgrade from 15.0.